.github/workflows/DefaultCache.yml

```diff
@@ -0,0 +1,20 @@
+name: DefaultCache
+
+on:
+  push:
```

Comment:
This should only run when master is updated, so it isn't checked until it is merged.
But I tried it with 'pull_request' to verify that everything builds.